activesupport/test/hash_with_indifferent_access_test.rb

```diff
@@ -527,7 +527,7 @@ def test_indifferent_hash_with_array_of_hashes
     hash = { "urls" => { "url" => [ { "address" => "1" }, { "address" => "2" } ] } }.with_indifferent_access
     assert_equal "1", hash[:urls][:url].first[:address]
 
-    hash = hash.to_hash
+    hash = hash.to_hash # rubocop:disable Style/RedundantSelfAssignment
```

Comment:
Seems like this is a false positive?

Reply:
Yeah, this is a false positive. 
I will fix this in upstream rubocop. Wdyt about leaving this as inline for now?

Reply:
My concern is that any new code similar to this would also have to add the pragma to keep Rubocop happy.  And we don't know exactly when Rubocop will be fixed, released, and updated in Rails, so my opinion is that it's best to remove it until Rubocop is fixed.